### ISSUE TITLE
[Feature request] a better README.md

### ISSUE THREAD

**Issue by zetok** [pcwalton/libui-rs#7]
- [ ] how to build
- [ ] should clearly state that it's dual-licensed, just putting both licenses in repo isn't really meaningful
- [ ] does it even compile on stable?

**Comment by zeyonaut** [comment]
In addition to that, stating the requirements for contributing both legal and other, and some screenshots/example code to get new users started.

**Comment by snuk182** [comment]
+ is it ever possible to build it on Windows, both with GNU & MSVC.
  UPD possible, but with adventures.

**Comment by faraazahmad** [comment]
Is this project still active?

**Comment by mitchhentges** [comment]
No, but one of the strongest forks is [`leotindall`'s](https://github.com/LeoTindall/libui-rs/)
